simplecpp.cpp

```diff
@@ -1026,22 +1026,22 @@ namespace simplecpp {
          * @param output   destination tokenlist
          * @param rawtok   macro token
          * @param macros   list of macros
-         * @param files    the files
+         * @param inputfiles    the files
```

Comment:
I use the name "files" everywhere for this. I'd say it's good to use the same name so it can be seen it is the same data.